### PR TITLE
my code for login screen

### PR DESCRIPTION
**Pull request by Fajeeer**

### CODE REVIEW

File: frontend-react/package.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "mobx": "^5.15.6",
+    "mobx-react": "^6.3.0",
+    "modulo": "0.0.3",
```
Comment (michaeliwaikato) [inline frontend-react/package.json]: Not really sure what these are used for or if they are even necessary?

File: frontend-react/src/components/landingpage.js [hunk 2]
```diff
@@ -1,12 +1,58 @@
 import React, { Component } from 'react';
+import Navigation from '../navigation.js';
```
Comment (michaeliwaikato) [inline frontend-react/src/components/landingpage.js]: Is there any particular reason this is here?

File: frontend-react/src/components/landingpage.js [hunk 3]
```diff
@@ -1,12 +1,58 @@
...
 import '../style/landingpage.css';
 
 class LandingPage extends Component {
+    handleClick() {
+        console.log('Click happened');
+    }
```
Comment (michaeliwaikato) [inline frontend-react/src/components/landingpage.js]: This code is not being used so it shouldn't be here.

File: frontend-react/src/components/landingpage.js [hunk 4]
```diff
@@ -1,12 +1,58 @@
...
+                            <input id={'username'}></input>
+                        </div>
+
+                         <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginRight: "0.1rem"}} for={'password'}>Password</label>
+                            <input id={'password'} type={'password'}></input>
+                        </div>
+
+                        <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginRight: "2.5rem"}} for={'port'}>Port</label>
```
Comment (michaeliwaikato) [inline frontend-react/src/components/landingpage.js]: Styling should be applied in style/landingpage.css
The source images should also be given a better name and be put in a new folder such as "resources" or something like that.

File: frontend-react/src/components/landingpage.js [hunk 5]
```diff
@@ -1,12 +1,58 @@
...
+const styles = {
+    rectangle: {
+        width: '500px',
+        height: '170px',
+        padding: '50px',
+        marginLeft: '70px',
+        backgroundColor: 'white',
+        borderRadius: 10
+    }
+}
```
Comment (michaeliwaikato) [inline frontend-react/src/components/landingpage.js]: Same as above, this should be in style/landingpage.css

**Review by michaeliwaikato — CHANGES_REQUESTED** [review]

File: frontend-react/src/components/landingpage.js [hunk 6]
```diff
@@ -1,12 +1,42 @@
...
     render() {
-        return (
-            <div><h1 className="code">Landing Page</h1></div>
+        return (   
+            <div style={{display: "flex"}}>
+                <div className="form">
+                    <img className="logo" src={Logo}></img>
+                </div>
+                <form calssName="form">
+                    <h1 className="head1">User Friendly and nbsp</h1>
```
Comment (andrei-msd) [inline frontend-react/src/components/landingpage.js:15]: The h1 should be "User Friendly Deep Learning"

Comment (andrei-msd) [inline frontend-react/src/components/landingpage.js]:
```suggestion
                    <h1 className="head1">User Friendly</h1>
```

**Review by andrei-msd — CHANGES_REQUESTED** [review]

**Review by andrei-msd — APPROVED** [review]

**Pull request merged**